Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/cheats/ui/CheatListFragment.java

```diff
@@ -16,24 +16,27 @@
 
 import com.google.android.material.divider.MaterialDividerItemDecoration;
 
-import org.dolphinemu.dolphinemu.R;
+import org.dolphinemu.dolphinemu.databinding.FragmentCheatListBinding;
 import org.dolphinemu.dolphinemu.features.cheats.model.CheatsViewModel;
 import org.dolphinemu.dolphinemu.utils.InsetsHelper;
 
 public class CheatListFragment extends Fragment
 {
+  private FragmentCheatListBinding mBinding;
```

Comment:
Is this needed?

Reply:
Yes, so we can set the binding to null when the fragment is destroyed.

Reply:
I think I missed the reference to `mBinding.cheatList`.